news/issues/issue1.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: single
+title:  "*Flying*  Issue 1"
+sidebar:
+  nav: news
+---
+
+
+## Dear Spinnaker enthusiasts,
+
+Hi! Welcome to the Spinnaker Newsletter, your source for info about the project and community. We'll focus on news for Spinnaker operators, developers, and contributors.
```

Comment:
How does someone get their news/content into the newsletter? Is it relying on you or someone else finding it? Is there a pathway for people to submit content?

Reply:
This is a good question @brian-armory and I think right now, the answer is, they can submit it as a comment or issue, or can submit their own PR? Or yeah, contact me? It's probably good to outline that. But I will note, the kubelist doesn't have any mechanism like that. the CNCF marketing team or whatever curates the content. So I suppose to get your content added there you just send it to them? I'd be happy to set up an email account or something dedicated to this and I could check it before each issue or something

Reply:
So far the only folks who've cared are OpsMX, and I emailed them about this PR and suggested that they comment on it if they wanted to add content